docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -853,7 +853,7 @@ Default templates for project boards:
   - `ResponseWriter`: the responseWriter from the request.
   - `RequestID`: the value matching REQUEST_ID_HEADERS（default: `-`, if not matched）.
   - You must be very careful to ensure that this template does not throw errors or panics as this template runs outside the panic/recovery script.
-- `REQUEST_ID_HEADERS`: **_empty_**: You can configure multiple values that are splited by comma here. It will match in the order of configuration, and the first match will be finally printed in the access log.
+- `REQUEST_ID_HEADERS`: **_empty_**: You can configure multiple values that are split by comma here. It will match in the order of configuration, and the first match will be finally printed in the access log.
```

Comment:
`splitted` or `separated`?

Reply:
https://livexp.com/blog/is-it-split-or-splitted-and-some-other-similar-irregular-verbs/?utm_source=blog-en&utm_medium=article&utm_campaign=%2Fblog%2Fis-it-split-or-splitted-and-some-other-similar-irregular-verbs%2F so `splitted` is incorrect in the context of the past `split` - `split` is correct.